qutebrowser/config/configdata.yml

```diff
@@ -1173,6 +1173,44 @@ editor.encoding:
   default: utf-8
   desc: Encoding to use for the editor.
 
+## fileselect
+
+fileselect.handler:
+  default: default
+  type:
+    name: String
+    valid_values:
+      - default: "Use the default file selector."
+      - external: "Use an external command."
+  desc: Handler for selecting file(s) in forms.
```

Comment:
Maybe add something pointing to `fileselect.single_file.command` and `fileselect.multiple_files.command` here - see https://github.com/qutebrowser/qutebrowser/pull/4477#discussion_r251257562 (I'd probably add it in the setting description here, rather than in the value description above).

Reply:
Updated description to point to `fileselect.single_file.command` and `fileselect.multiple_files.command`